### PR TITLE
Update demo application 2.1 from mapfish-print 3.6.0 to 3.10.2

### PR DESCRIPTION
**Pull request by jwkaltz**

### CODE REVIEW

**Comment by jwkaltz** [comment]
@pvalsecc can you please check if these are the right steps to upgrade mapfish-print for our demo application GMF 2.1?
I did try this is out on our test server, it seems to work fine.
FYI we also started the following page for instructions on how to upgrade: https://confluence.camptocamp.com/confluence/display/GS/Update+MapFish+Print+to+3.10.2

If these changes are correct, I will also make PRs for the other demo versions and for c2cgeoportal as well.

**Comment by jwkaltz** [comment]
@pvalsecc I made more changes (clean up of the CONST_Makefile) -> can you please check again?

File: CONST_Makefile [hunk 1]
```diff
@@ -1144,24 +1141,6 @@ ifneq ($(TOMCAT_START_COMMAND),)
...
-
-print/WEB-INF/lib/postgresql-9.3-1102.jdbc41.jar:
-	$(PRERULE_CMD)
-	mkdir -p $(dir $@)
-	curl --max-redirs 0 --location --output $@ https://jdbc.postgresql.org/download/postgresql-9.3-1102.jdbc41.jar
-	unzip -t -q $@
-
 $(VENV_BIN)/generate_controller$(PYTHON_BIN_POSTFIX): .build/dev-requirements.timestamp
 
 # Tile cloud chain
```
Comment (pvalsecc) [inline CONST_Makefile]: You are not supposed to change the files starting by `CONST_` This needs to be fixed in c2cgeoportal instead:
https://github.com/camptocamp/c2cgeoportal/blob/master/c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl#L436-L439

**Comment by pvalsecc** [comment]
Travis fails: https://travis-ci.org/camptocamp/demo_geomapfish/jobs/274149095#L974

**Comment by jwkaltz** [comment]
I guess it is simpler to wait for a new version of c2cgeoportal and an update to demo project based on that version -> so closing this PR.

**Pull request closed without merging**